lib/slime/parser.ex

```diff
@@ -200,7 +201,15 @@ defmodule Slime.Parser do
     if parts["leading_space"] != "", do: spaces = Dict.put(spaces, :leading, true)
     if parts["trailing_space"] != "", do: spaces = Dict.put(spaces, :trailing, true)
 
-    {tag, css_classes(parts["css"]) ++ html_id(parts["id"]), spaces}
+    case Regex.named_captures(@id_regex, parts["css"]) do
+      nil ->
+        {tag, css_classes(parts["css"]) ++ html_id(""), spaces}
+      capture ->
+        IO.inspect(capture["id"])
+        IO.inspect(parts["css"])
+        css_classes = String.replace(parts["css"], "##{capture["id"]}", "")
```

Comment:
@lpil dogma doesn't like it => https://travis-ci.org/slime-lang/slime/jobs/116487392#L414